commit 4c717f20ae5630557ce8dfee8bd31df0c3d80a25
Author: Sylvestre Ledru <sylvestre.ledru@scilab.org>
Date:   Thu Feb 4 16:59:13 2010 +0100

    * bug 5496 fixed - When the option --disable-build-help was enabled, the
                       check/use of jeuclid was disabled. It was causing build
                       issue since jeuclid is now necessary for graphics.

diff --git a/scilab/configure.ac b/scilab/configure.ac
index 4e53006..71c39af 100644
--- a/scilab/configure.ac
+++ b/scilab/configure.ac
@@ -749,6 +749,14 @@ if test "$with_javasci" != no -o "$with_gui" != no -o "$enable_build_help" != no
 				JROSETTA_ENGINE=$PACKAGE_JAR_FILE
 				AC_SUBST(JROSETTA_ENGINE)
 
+
+				# MathML rendering solution
+				# Used in both graphic & help 
+				AC_JAVA_CHECK_PACKAGE([jeuclid-core],[net.sourceforge.jeuclid.LayoutContext],[MathML rendering solution (at least version 3.1.X)])
+				JEUCLID_CORE=$PACKAGE_JAR_FILE
+				AC_SUBST(JEUCLID_CORE)
+
+
 				if test $HDF5_ENABLE = yes;  then
 				# HDF5 java lib
 				  AC_JAVA_CHECK_PACKAGE([jhdf5],[ncsa.hdf.hdf5lib.HDF5Constants],[HDF5 Java library])
@@ -1772,7 +1780,6 @@ if test $HELP_ENABLE = yes; then
    echo "  Batik (SVG) ............ = $BATIK"
    echo "  Saxon XSLT ............. = $SAXON"
    echo "  FOP (XML => PDF) ....... = $FOP"
-   echo "  Jeuclid (MathML) ....... = $JEUCLID_CORE"
    echo "  Commons I/O ............ = $COMMONS_IO"
    echo "  XML graphics commons ... = $XMLGRAPHICS_COMMONS"
    echo "  XML commons external ... = $XML_APIS_EXT"
@@ -1807,6 +1814,7 @@ echo "  JOGL ................ = $JOGL"
 echo "  JOGL LIBS (JNI) ..... = $JOGL_LIBS"
 echo "  Gluegen ............. = $GLUEGEN_RT"
 echo "  Gluegen LIBS (JNI) .. = $GLUEGEN_RT_LIBS"
+echo "  Jeuclid (MathML) .... = $JEUCLID_CORE"
 echo "  Jhall  .............. = $JHALL"
 echo "  Jrosetta (API) ...... = $JROSETTA_API"
 echo "  Jrosetta (Engine) ... = $JROSETTA_ENGINE"
diff --git a/scilab/m4/docbook.m4 b/scilab/m4/docbook.m4
index e017774..7fbbcca 100644
--- a/scilab/m4/docbook.m4
+++ b/scilab/m4/docbook.m4
@@ -54,11 +54,6 @@ AC_ARG_WITH(docbook,
 	FOP=$PACKAGE_JAR_FILE
 	AC_SUBST(FOP)
 
-	# MathML rendering solution
-	AC_JAVA_CHECK_PACKAGE([jeuclid-core],[net.sourceforge.jeuclid.LayoutContext],[MathML rendering solution (at least version 3.1.X)])
-	JEUCLID_CORE=$PACKAGE_JAR_FILE
-	AC_SUBST(JEUCLID_CORE)
-
 	# Commons I/O library
 	AC_JAVA_CHECK_PACKAGE([commons-io],[org.apache.commons.io.output.CountingOutputStream],[Commons I/O library])
 	COMMONS_IO=$PACKAGE_JAR_FILE
